prometheus-ingestion-retrieval.md

```diff
@@ -0,0 +1,67 @@
+# Prometheus Ingestion
+
+IRONdb has native endpoints for accepting remote write data from a Prometheus
+installation.  Once the Prometheus module is enabled, data can be send to IRONdb 
+by setting the Prometheus remote_write endpoint to:
+
+`http://irondbnode:8112/module/prometheus/write/<accountid>/<uuid>`
+
+## Enabling Prometheus Module
+
+IRONdb must be [configured](configuration.html) such that the Prometheus module is
+enabled for reading or writing Prometheus data natively.  In the <modules> section of 
+the IRONdb config, Prometheus support is activated by including the following stanza:
+```
+<generic image="prometheus" name="prometheus"/>
+```
+Depending on installation this may be shipped enabled by default.
+
+## Namespacing
+
+Prometheus data is not namespaced by nature.  This can create confusion if different
+copies of Prometheus have identically named metrics.  Inside of IRONdb, we require 
+that all data be namespaced under a UUID.  This UUID can be arbitrarily chosen or 
+created using `uuidgen` on a typical unix system.  Each distinct set of 
+Prometheus data should have it's own UUID.   For high-availability in Prometheus it
+is the recommended pratice to have two copies collecting the same data.  While these
+two instances do not contain the same data, they do represent the same metrics, and so
+should share a common UUID for their namespace.  You may wish to send both of these
+nodes into IRONdb, where they will get merged into simply being more samples for the
+particular metrics time-streams.
```

Comment:
The last sentence reads awkwardly. Perhaps something like, "where they simply become more samples in the given metric stream"?

Also, I assume "nodes" is a Prometheus term for a metric stream?